filters/src/main/java/com/matecat/converter/core/XliffBuilder.java

```diff
@@ -54,7 +54,7 @@
 public class XliffBuilder {
 
     // Logger
-    private static Logger LOGGER = LoggerFactory.getLogger(XliffBuilder.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(XliffBuilder.class);
```

Comment:
@AlexFalappa why you choose to have static fields as loggers?

Reply:
The loggers were already static I made them final to prevent unintented modifications.

Having private static final loggers is a common practice so there will be one logger object instance per class shared by all objects of that class (loggers are inherently thread safe).

The only case where I choose not to have static loggers is in object hierarchies when the logger object is declared in the parent class, in that case it is better to have it non static so that it is different in each derived class.

Reply:
Ok